subdivision.scad

```diff
@@ -28,3 +28,113 @@ function subdivide_faces(n=1, poly) =
     ])
   )
   n > 1 ? subdivide_faces(n-1, [allpoints, faces]) : n > 0 ? [allpoints,faces] : poly;
+
+//Utility function for splitting circular vectors.
+function range (list,start,end)=
```

Comment:
Also, maybe this should go in "utilities" section of functional.scad.  
Do you think it would be useful in other circumstances, or do you feel its just a very niche thing that would not be needed again?

Reply:
It seems like a general thing, but I don't have a lot of experience developing this kind of thing.